patternfly/patternfly-elements
# docs(badge): updated Badge page

## <Pull request description>




### Related issues

- #2217 


### Preview


- [Preview of the Badge page](https://deploy-preview-2218--patternfly-elements.netlify.app/components/badge/)
- [Preview of the All Components page](https://deploy-preview-2218--patternfly-elements.netlify.app/components/)


### What has changed and why



- Removed examples and code for`info`, `success`, `moderate`, `important`, `critical` states to docs
- Added examples and code for `read` and `unread` states
- Updated badge image on the All Components page


### Testing instructions



- View [preview of the Badge page](https://deploy-preview-2218--patternfly-elements.netlify.app/components/badge/)
- View [preview of the All Components page](https://deploy-preview-2218--patternfly-elements.netlify.app/components/) to see updated image for badge
